src/NetAnalyzers/UnitTests/Microsoft.CodeQuality.Analyzers/Maintainability/ReviewUnusedParametersTests.Fixer.cs

```diff
@@ -511,6 +511,18 @@ class MyValue
             await VerifyCS.VerifyCodeFixAsync(code, fix);
         }
 
+        [Fact]
+        public async Task Operators_CSharp()
+        {
+            var code = @"
+public struct Unit
+{
+    public static bool operator ==(Unit [|x|], Unit [|y|]) => true;
+    public static bool operator !=(Unit [|x|], Unit [|y|]) => false;
+}";
+            await VerifyCS.VerifyCodeFixAsync(code, code);
+        }
```

Comment:
This test is failing with:

```
    Microsoft.CodeAnalysis.Testing.Verifiers.EqualWithMessageException : Context: Iterative code fix application
    Expected '0' iterations but found '1' iterations.
```

Not sure what this means or how to fix that.

AFAIK, `VerifyCS.VerifyCodeFixAsync(code, code)` verifies the diagnostics and verifies that no code fix is offered.

Reply:
Ok after reading the fixer code, it sounds like this is *an existing bug*, where a codefix is offered but it does "nothing":

![image](https://user-images.githubusercontent.com/31348972/104755516-335c5500-5763-11eb-81c8-a8e3c1e44650.png)